webapi/src/test/scala/org/knora/webapi/e2e/v2/StandoffRouteV2E2ESpec.scala

```diff
@@ -0,0 +1,355 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.e2e.v2
+
+import java.nio.file.Paths
+import akka.http.javadsl.model.StatusCodes
+import akka.http.scaladsl.model.headers.BasicHttpCredentials
+import akka.http.scaladsl.model.{HttpEntity, _}
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import org.knora.webapi.SipiUploadResponseJsonProtocol.sipiUploadResponseFormat
+import org.knora.webapi._
+import org.knora.webapi.e2e.v2.ResponseCheckerV2.compareJSONLDForMappingCreationResponse
+import org.knora.webapi.messages.OntologyConstants
+import org.knora.webapi.messages.store.triplestoremessages.RdfDataObject
+import org.knora.webapi.messages.util.rdf.{JsonLDDocument, JsonLDKeywords}
+import org.knora.webapi.messages.v2.routing.authenticationmessages.{AuthenticationV2JsonProtocol, LoginResponse}
+import org.knora.webapi.sharedtestdata.SharedTestDataADM
+import org.knora.webapi.sharedtestdata.SharedTestDataV1.ANYTHING_PROJECT_IRI
+import org.knora.webapi.util.{FileUtil, MutableTestIri}
+import org.knora.webapi.messages.StringFormatter
+import org.knora.webapi.models.filemodels.{FileType, UploadFileRequest}
+import org.knora.webapi.models.standoffmodels.DefineStandoffMapping
+import org.xmlunit.builder.{DiffBuilder, Input}
+import org.xmlunit.diff.Diff
+import spray.json._
+
+import java.net.URLEncoder
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+/**
+ * End-to-end test specification for the search endpoint. This specification uses the Spray Testkit as documented
+ * here: http://spray.io/documentation/1.2.2/spray-testkit/
```

Comment:
I think this docstring has to be adjusted. It mentions the search endpoint which is not used here.

Reply:
yes... typical copy-paste